sar/WFLY-13352-Add_galleon_layer_sar_subsystem.adoc

```diff
@@ -0,0 +1,100 @@
+= Galleon layer for SAR subsystem
+:author:            Yeray Borges
+:email:             yborgess@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The SAR subsystem allows to deploy .sar archives containing MBeans configured by META-INF/jboss-service.xml.
+
+The goal of this feature is to supply a https://docs.wildfly.org/galleon/#_layers[Galleon layer] which will be responsible for providing the SAR subsystem.
+
+The following is a configuration example of the WildFly bootable JAR Maven plugin to provision a server with the `sar` layer:
+
+[source,xml]
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-jar-maven-plugin</artifactId>
+    <configuration>
+        <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#${version.wildfly}</feature-pack-location>
+        <layers>
+            <layer>sar</layer>
+        </layers>
+    </configuration>
+    <executions>
+        <execution>
+            <goals>
+                <goal>package</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+----
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-13352[WFLY-13352 Add a Galleon layer for the SAR subsystem]
+* https://issues.redhat.com/browse/EAP7-1483[EAP7-1483 Add Galleon layers for remaining non-deprecated subsystems]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+WildFly
+
+=== Other Interested Projects
+
+N/A
+
+== Requirements
+
+* The Galleon layer must provide the sar subsystem.
+* Since the sar subsystem uses services configured by the JMX subsystem, we need to depend on JMX Galleon layer which supplies the JMX subsystem.
```

Comment:
This is just an internal dependency right?
I mean the user is free not to have JMX layer in the config, correct?

Reply:
Correct.

The JMX layer is an internal dependency on the SAR Galleon layer. If a user is provisioning SAR Galleon layer, the JMX will be brought in automatically as a transitive dependency.

Reply:
ok thanks